lib/membrane/rtp/packet.ex

```diff
@@ -45,13 +48,37 @@ defmodule Membrane.RTP.Packet do
     end
   end
 
+  defp serialize_header_extensions([]), do: <<>>
+
+  defp serialize_header_extensions(extensions) do
+    extensions =
+      Enum.reduce(extensions, <<>>, fn extension, acc ->
+        acc <> serialize_header_extension(extension)
+      end)
+
+    identifer = @one_byte_header_identifier
+    padding = calculate_padding_size(byte_size(extensions)) * 8
+    extensions_size = byte_size(extensions)
+
+    extension_header_size =
+      div(extensions_size, 4) + if rem(extensions_size, 4) == 0, do: 0, else: 1
+
+    <<identifer::binary-size(2), extension_header_size::16>> <>
+      extensions <> <<0::integer-size(padding)>>
```

Comment:
can you put that all into a single binary? Erlang then can optimize the call and right now it will probably create 3 diferent binaries and concatenate them.